pocs/tests/test_observatory.py

```diff
@@ -388,9 +391,11 @@ def test_no_dome(observatory):
     assert observatory.close_dome()
 
 
-def test_operate_dome(config_with_simulated_dome):
+def test_operate_dome(config_with_simulated_dome, config):
```

Comment:
This is a little weird to me but appears to be working. I think we could just have one config. What happens if you remove the `config_with_simulated_dome` and just use the `config`? @jamessynge might have some insight with the simulated dome.